### PR TITLE
TA- 635-Issue direction cross browser tests

### PR DESCRIPTION
**Pull request by phaniperla**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/TA-635

### Change description ###

Issue Decision Cross browser test

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: e2e/features/issue-decision.feature [hunk 1]
```diff
@@ -15,7 +15,7 @@ Feature: Issue decision
     And I navigate to an existing case
     When I choose "Write final decision"
 
-  @issue-decision @nightly-test
+  @issue-decision @nightly-test @crossbrowser99
```
Comment (gokul-sol) [inline e2e/features/issue-decision.feature:18]: @phaniperla  should this be @crossbrowser ?

Comment (phaniperla) [reply e2e/features/issue-decision.feature:18]: There is a known issue in sauce labs wrt to remote file uploading and no workaround solution available so therefore that scenario has been excluded from  cross browser.

Comment (phaniperla) [reply e2e/features/issue-decision.feature:18]: Please refer below JIRA ticket for further details

Comment (phaniperla) [reply e2e/features/issue-decision.feature:18]: https://tools.hmcts.net/jira/browse/TA-636

**Review by gokul-sol — APPROVED** [review]

**Pull request merged**